pkg/cmd/issue/shared/lookup.go

```diff
@@ -41,6 +41,38 @@ func IssueFromArgWithFields(httpClient *http.Client, baseRepoFn func() (ghrepo.I
 	return issue, baseRepo, err
 }
 
+// IssuesFromArgWithFields loads 1 oe more issues or pull requests with the specified fields. If some of the fields
+// could not be fetched by GraphQL, this returns non-nil issues and a *PartialLoadError.
+func IssuesFromArgsWithFields(httpClient *http.Client, baseRepoFn func() (ghrepo.Interface, error), args []string, fields []string) ([]*api.Issue, ghrepo.Interface, error) {
+	if len(args) == 0 {
+		return nil, nil, fmt.Errorf("missing required issue number")
+	}
+
+	issues := make([]*api.Issue, 0, len(args))
+	var issuesRepo ghrepo.Interface
+
+	for _, arg := range args {
```

Comment:
It would be nice to also make the retrieval of the issues parallelized. What do you think?

Reply:
Yeah, but I'll use an `errgroup.Group` here because I wouldn't expect any 1 fetch to fail and continue with the others. Ideally, it'd be great to fetch all issues in one request, but seems there's no REST or GraphQL endpoint capable of doing such. Might be able to use aliasing with GraphQL but it's quite a hack and probably not worth it. I'd be surprised if people are updating enough issues to really make it worth it.

Reply:
That makes sense to me to use the `errgroup.Group` here. Agreed, lets not try to optimize for a single request right now, seems like even if it was possible it would be a pretty big refactor.